### PR TITLE
[SDK-2057] Removed v1/event code

### PR DESCRIPTION
**Pull request by nsingh-branch**
## Reference
SDK-2057 -- Removed v1/event code

## Summary
Removed `userCompletedAction` and `sendCommerceEvent` code from `RNBranchModule.java`, `RNBranch.m`, `branchUniversalObject.js`, and `index.js`.

## Motivation
Remove old code that has been deprecated for a sufficient amount of time.

## Type Of Change

- [x] Bug fix (non-breaking change which fixes an issue)

## Testing Instructions

Integrate the SDK, run the app, and run the tests to ensure everything is still functioning correctly.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

File: src/branchUniversalObject.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ export default async function createBranchUniversalObject(identifier, options =
 
       return this._tryFunction(RNBranch.showShareSheet, shareOptions, linkProperties, controlParams)
     },
-    // deprecated in favor of userCompletedAction(RegisterViewEvent)
+    // deprecated in favor of BranchEvent
     registerView() {
```
Comment (gdeluna-branch) [inline src/branchUniversalObject.js:70]: Does this comment imply we should remove `registerView` as well? It's just an alias to a `BranchEvent.logEvent` underneath

File: src/branchUniversalObject.js [hunk 2]
```diff
@@ -79,13 +79,6 @@ export default async function createBranchUniversalObject(identifier, options =
       if (Platform.OS !== 'ios') return Promise.resolve()
       return this._tryFunction(RNBranch.listOnSpotlight)
```
[the remark this answers is not in the document]
Comment (JagadeeshKaricherla-branch) [reply src/branchUniversalObject.js:80]: I think the below deprecated code can now be safely deleted :

https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/src/branchUniversalObject.js#L30

https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/src/branchUniversalObject.js#L70

https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/src/branchUniversalObject.js#L77

**Comment by JagadeeshKaricherla-branch** [comment]
testbed app has some of these deprecated methods(now removed) : 

https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/branchreactnativetestbed/App.tsx#L137-L138

**Comment by nsingh-branch** [comment]
> testbed app has some of these deprecated methods(now removed) :
> 
> https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/branchreactnativetestbed/App.tsx#L137-L138

The name is bit misleading, but this function has been updated to call a standard Branch Purchase event instead of sendCommerceEvent already.

**Comment by JagadeeshKaricherla-branch** [comment]
> > testbed app has some of these deprecated methods(now removed) :
> > https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/branchreactnativetestbed/App.tsx#L137-L138
> 
> The name is bit misleading, but this function has been updated to call a standard Branch Purchase event instead of sendCommerceEvent already.

ok I see : https://github.com/BranchMetrics/react-native-branch-deep-linking-attribution/blob/0ccf24170d25739c749a44c69f09003dbbe727b2/branchreactnativetestbed/components/BranchWrapper.ts#L219-L220

File: src/branchUniversalObject.js (hunk 1 above)
Comment (nsingh-branch) [reply src/branchUniversalObject.js:70]: On the iOS side, I believe there is still some code that works with Spotlight so I don't think we can fully remove it yet.

**Review by gdeluna-branch — APPROVED** [review]

**Pull request merged**